### DISCUSSION TITLE
high availability In 1.29.2

### DISCUSSION THREAD

**Discussion by chenxidong** [zendesk/maxwell#2138] [General]
In 1.29.2, is high availability still supported through jgroups-raft? Has the configuration changed?

**Comment by osheroff** [comment]
Nothing has changed as far as I know. On Dec 14, 2024, at 09:40, chenxidong ***@***.***> wrote:﻿
In 1.29.2, is high availability still supported through jgroups-raft? Has the configuration changed?

**Comment by chenxidong** [comment]
这是来自陈喜栋QQ邮箱的自动回复邮件。
 
您好，您的邮件我已收到，我会尽快给您回复，谢谢！
